.github/workflows/release.yml

```diff
@@ -0,0 +1,17 @@
+name: Perform actions after release
+
+on:
+  release:
+    type: [published, created, edited]
+
+jobs:
+  release_image:
+    name: Build and publish to Docker Hub
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@master
```

Comment:
Should it be master? because I can make a tag, then add few commits to master and then make a release from that old tag

Reply:
1. In this line `master` is reference to action repo branch, i.e. `checkout master branch from https://github.com/actions/checkout`, not `checkout master from https://github.com/kiwicom/schemathesis/`.
Maybe it is better to use release tags, e.g. `actions/checkout@v1` but this is the same as in `build.yml`

To specify specific branch/tag for this repo syntax would be:
```
- uses: actions/checkout@v1
  with:
    ref: some-branch
```

More here: https://help.github.com/en/actions/automating-your-workflow-with-github-actions/workflow-syntax-for-github-actions#example-using-versioned-actions

2. By default checkout action checkouts specific commit, which hash is stored in `GITHUB_SHA` and it depends on event that triggered job. Here it is triggered by release, release always is associated with tag and in this case `GITHUB_SHA` is last commit in the tagged release.

More info: https://help.github.com/en/actions/automating-your-workflow-with-github-actions/events-that-trigger-workflows#release-event-release